packages/experimental/Avatar/src/__tests__/JSAvatar.test.jsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { JSAvatar } from '..';
+import { getInitials, validateAlphabeticalCharacters, removeTitlesFromName } from '../useAvatar';
+import * as renderer from 'react-test-renderer';
+
+const alphabeticalTestNames = [
+  'Marie',
+  'Aadi KaPoor',
+  'Marie Beaudouin',
+  'Aadi Meni KaPoor',
+  'Aadi Meni Ka Jr',
+  'Will Little (Teams)',
+  'Will Little-Tanaka',
+  '-Aadi Kapoor',
+  'Richard 23 Feynman',
+  '* Richard *',
+  'Richard Feynman [Teams]',
+  'Richard Feynman {Teams}',
+];
+const alphabeticalInitials = ['M', 'AK', 'MB', 'AK', 'AK', 'WL', 'WL', 'AK', 'RF', 'R', 'RF', 'RF'];
+const otherTestNames = ['(206) 123-4567', '耿盈盈']; // will return this case after bug bash - '耿 James'
+const namesWithTitles = ['Mr Feynman', 'Dr Richard Feynman'];
+const initialsForNameWithTitles = ['F', 'RF'];
+
+it('renders Avatar', () => {
```

Comment:
Should this be wrapped in a "describe"?

Reply:
Yeah, I added this change. It'll be it today's PR :)